diff --git a/node_modules/immer/dist/immer.d.ts b/node_modules/immer/dist/immer.d.ts
index 0000000..0000000 100644
--- a/node_modules/immer/dist/immer.d.ts
+++ b/node_modules/immer/dist/immer.d.ts
@@ -208,10 +208,6 @@ declare function current<T>(value: Draft<T>): T;
 
 declare function enablePatches(): void;
 
-declare global {
-    var Iterator: undefined | {
-        from<T, TReturn>(iterable: Iterator<T, TReturn>): IterableIterator<T>;
-    };
-}
 declare function enableMapSet(): void;
